src/App.jsx

```diff
@@ -1,40 +1,40 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
+  const currentSelectedTab = tabs.find(tab => tab.id === selectedTabId);
 
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
+  return (
+    <div className="section">
+      <h1 className="title">
+        {`Selected tab is ${currentSelectedTab.title}`}
+      </h1>
 
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+      <div data-cy="TabsComponent">
+        <div className="tabs is-boxed">
+          <ul>
+            <Tabs
+              tabs={tabs}
+              selectedTabId={selectedTabId}
+              onTabSelected={tab => setSelectedTabId(tab.id)}
+            />
+          </ul>
+        </div>
 
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
-
-      <div className="block" data-cy="TabContent">
-        Some text 1
+        <div className="block" data-cy="TabContent">
+          {currentSelectedTab.content}
+        </div>
```

Comment:
```suggestion

```
Move this block to the tab component

Reply:
but where?
in that component only tabs, is i add it there if will appear right besides them